generators/deployment/templates/pipeline_master.yml

```diff
@@ -0,0 +1,192 @@
+stages:
+- name: Build Stage
+  inputs:
+  - type: git
+    branch: master
+    service: ${REPO}
+{{#if triggersType}}
+  triggers:
+    - type: {{triggersType}}
+{{/if}}
+  jobs:
+  - name: Build
+    type: builder
+{{#each buildJobProps}}
+    {{@key}} : {{{this}}}
+{{/each}}
+  {{#has deployment.type 'Kube'}}
+  - name: Container Build
+    type: builder
+    extension_id: ibm.devops.services.pipeline.container.builder
+    target:
+      region_id: ${CF_REGION_ID}
+      organization: ${CF_ORGANIZATION}
+      space: ${CF_SPACE}
+    IMAGE_NAME: {{deployment.name}}
+    USE_CACHED_LAYERS: 'true'
+    COMMAND: |
+      #!/bin/bash
```

Comment:
Could this be included in a separate script file rather than being in here? It makes this file very long